openapi/components/schemas/Subscription/OrderLineItem.yaml

```
@@ -1,10 +1,6 @@
 type: object
-description: Invoice line item.
-required:
-  - type
-  - unitPriceAmount
-  - unitPriceCurrency
-  - quantity
+description: Invoice line item. Use `isInterim` property of upcoming invoice items instead.
```

suggestion: suggestion for readability
```suggestion
description: Invoice line item. Use the `isInterim` property for upcoming invoice items.
```

---

What about?
```suggestion
description: Use the `isInterim` property from upcoming invoice items.
```
The point is to redirect from the deprecated field to its new alternative.

---

Looks good.